subgraphs/enzyme-core/contexts/ethereum.ts

```diff
@@ -129,6 +129,7 @@ export const ethereum: Context<Variables> = {
         minAssetBalancesPostRedemptionPolicyAddress: suluDeployment.contracts.MinAssetBalancesPostRedemptionPolicy,
         minMaxInvestmentPolicyAddress: suluDeployment.contracts.MinMaxInvestmentPolicy,
         minSharesSupplyFeeAddress: suluDeployment.contracts.MinSharesSupplyFee,
+        noDepegOnRedeemSharesForSpecificAssetsPolicyAddress: suluDeployment.contracts.NoDepegOnRedeemSharesForSpecificAssetsPolicy,
```

Comment:
Use the actual address, we are no longer using the import from the environment.

Reply:
done